src/selectedtests/task_mappings/create_task_mappings.py

```diff
@@ -1,20 +1,19 @@
 """Method to create the task mappings for a given evergreen project."""
 import re
 
+from concurrent.futures import ThreadPoolExecutor as Executor
 from re import match
+from tempfile import TemporaryDirectory
 from typing import Dict, List, Pattern, Set, Tuple
 
 from boltons.iterutils import windowed_iter
-from concurrent.futures import ThreadPoolExecutor as Executor
-from evergreen.api import Version, Build, Task, EvergreenApi
+from evergreen.api import Build, EvergreenApi, Task, Version
 from evergreen.manifest import ManifestModule
-from git import Repo, DiffIndex
-from structlog import get_logger
-from tempfile import TemporaryDirectory
-
+from git import DiffIndex, Repo
 from selectedtests.evergreen_helper import get_evg_project
 from selectedtests.git_helper import get_changed_files, init_repo
 from selectedtests.task_mappings.version_limit import VersionLimit
+from structlog import get_logger
```

Comment:
It looks like `isort` is treating the local library as a 3rd party library. Do you know if there is a way to tell it the difference?

Reply:
Adding the `known_first_party` config option seems to have solved this